### ISSUE TITLE
upgrade project (and CI) to unity 2020.2

### ISSUE THREAD

**Issue by Rossco1337** [Rossco1337/7demake#4]
out of all the features added in 2020.2, the one that everyone (including myself) seems most excited for is the crumb of inspector functionality:
![image](https://user-images.githubusercontent.com/840675/105620391-7ddd7180-5df4-11eb-8f5c-6975f6693a44.png)

let's see what stands in our way of getting that into the project.

### issues
- [gableroux docker builds for CI haven't been updated yet](https://hub.docker.com/r/gableroux/unity3d/tags?page=1&ordering=last_updated&name=2020.2). no hate whatsoever, this is a guy doing unity's job as a free community service so he's entitled (and encouraged) to take as long as he wants.
- as the above step implies, i can't even activate an license automatically (#1) for 2020.2 as the build for it isn't available. however, 2020.1 might work fine for the webgl compilation as i'm not using any 2020.2 specific functionality yet. i'd be happy to test this but...
- unity's dark mode no longer works in 2020.2! ![image](https://user-images.githubusercontent.com/840675/105620467-6c489980-5df5-11eb-9e8b-9e74eeb976cd.png)
i can't remember what i did to get dark mode working in 2019.3 without paying (maybe some kind of config hack?) but the official response is "[no, it should just work](https://forum.unity.com/threads/dark-theme-disabled-on-unity-2020-1-1f1-personal.963974/) now because [we made it free](https://support.unity.com/hc/en-us/articles/115004982743-How-can-I-change-my-Editor-theme-)" and nobody is really talking about it. so this is a big ol' [DWOMM™](https://acronyms.thefreedictionary.com/DWOMM) Exclusive Category 1 "Prod-Breaker". can't do a thing without my dark mode.

### the short version
- [x] wait for the docker builds to update
- [x] do the licensing dance again
- [x] fix dark mode

**Comment by Rossco1337** [comment]
turns out i fixed the dark mode issue just be reinstalling the editors through unity hub. i'm gonna try updating the project to the 2021 beta and it i can get it to build, i'll merge that and update the CI/CD to 2020 when the docker images are out.
